analyzer/codechecker_analyzer/analyzers/clangtidy/analyzer.py

```diff
@@ -165,13 +165,51 @@ def get_warnings(env=None):
         raise
 
 
+def _add_star_for_group(
+    checkers: Iterable[str],
```

Comment:
`checkers` and `all_checkers`? Since this is a single purpose function, how about we call this `enabled_checkers`, or `subset_checkers`?